src/functions-and-arrays.js

```diff
@@ -52,17 +141,17 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
-
+function uniquifyArray(arrayofWordsIterationFive) {
+  arrayofWordsIterationFive.forEach(word => {
+    console.log(arrayofWordsIterationFive.indexOf(word))        
```

Comment:
o `indexOf` retorna o índice da primeira ocorrencia da palavra, ou seja, se `word` for uma palavra repetida, o índice dela não vai ser o índice da palavra q estamos olhando agora.
Nesse caso aqui recomendo usar o `for i` mesmo, asism usamos o i como índice do elemento para poder comparar